corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -88,11 +92,31 @@ private static void verifyNamespaceAndTablename(String namespace, String tableNa
      */
     public int printTable(String namespace, String tablename) {
         verifyNamespaceAndTablename(namespace, tablename);
+        StringBuilder builder;
+        // RegistryTable is a special system table that need not be opened
```

Comment:
we're doing the same thing in the compactor, can the same issue happen there? (because then we can work around it for now, until the scoping fix is in)